src/reference/config/telemetry/exporter/exporter-otlp.md

```diff
@@ -0,0 +1,95 @@
+---
+shortTitle: otlp
+description: Zilla runtime otlp exporter
+category:
+  - Telemetry
+tag:
+  - Exporters
+---
+
+# Open Telemetry Exporter
+
+Zilla runtime Open Telemetry exporter
+
+```yaml {3}
+exporters:
+  otlp:
+    type: otlp
+    options:
+      interval: 30
+      signals:
+        - metrics
+      endpoint:
+        protocol: http
+        location: http://otlp-collector:4318/v1/metrics
+```
+
+## Configuration
+
+:::: note Properties
+
+- [options](#options) *
+  - [options.interval](#options-interval)
+  - [options.signals](#options-signals)
+  - [options.endpoint](#options-endpoint) *
+    - [endpoint.protocol](#endpoint-protocol)
+    - [endpoint.location](#endpoint-location) *
+
+::: right
+\* required
+:::
+
+::::
+
+### options
+
+> `object`
+
+`otlp`-specific options.
+
+```yaml
+options:
+  interval: 30
+  signals:
+    - metrics
+  endpoint:
+    protocol: http
+    location: http://otlp-collector:4318/v1/metrics
+```
+
+#### options.interval
+
+> `integer`
+
+Interval in seconds to push data to the Open Telemetry collector. Default: 30 seconds.
+
+#### options.singals
```

Comment:
```suggestion
#### options.signals
```

Reply:
This will fix the link checker that failed on verifying the hash links in the table of contents.